### PR TITLE
Disabled auto mounting of default SA token for Frontend & MongoDB

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

#### What this PR does / why we need it:

**This PR will disable auto mounting of default SA token for Frontend & MongoDB**

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] DCO signed
- [X] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: charts/litmus/templates/frontend-deployment.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ spec:
           {{ toYaml .Values.portal.frontend.customLabels | nindent 8 }}
         {{- end }}
     spec:
+      automountServiceAccountToken: false
```
Comment (Jasstkn) [inline charts/litmus/templates/frontend-deployment.yaml:30]: should we make it configurable via Values?

Comment (rajdas98) [reply charts/litmus/templates/frontend-deployment.yaml:30]: @Jonsy13 maybe we can make it configurable and set default it to false.

Comment (rajdas98) [reply charts/litmus/templates/frontend-deployment.yaml:30]: It may vary in another system. All helm fields should be configurable(as a ideal case)

**Review by Jasstkn — APPROVED** [review]

**Comment by Jonsy13** [comment]
DO NOT MERGE FOR NOW. WILL TAKE THIS LATER

**Review by ksatchit — APPROVED** [review]

**Comment by ksatchit** [comment]
> **DO NOT MERGE** FOR NOW. WE WILL TAKE THIS PR LATER

Merging based on discussions with Author!

**Pull request merged**